### ISSUE TITLE
NTR: Dengue virus RT-PCR in blood or CSF positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8068]
New term request Dengue virus RT-PCR in blood or CSF positive (samant_101231001436): Cerebrospinal fluid; reverse transcriptase polymerase chain reaction; RNA  UMLS: 1540994

**Comment by pnrobinson** [comment]
@MickeySegal
if a separate term for CSF is needed, please open a new request.

Adding new term: 
- ID: HP:6000551
- label: Positive Dengue virus PCR test in the blood circulation
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: PMID:1372617
- def: Detection of nucleic acid of the Dengue virus in the blood circulation by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: Dengue viruses (family Flaviviridae, genus Flavivirus) occur as four antigenically distinct serotypes. Infection with any of them generally leads to a mild, self-limiting febrile illness (dengue fever). However, a more severe form of the disease, involving vascular and hemostatic abnormalities (dengue hemorrhagic fever-dengue shock syndrome), is responsible for a high mortality rate, primarily among children.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8068
- synonyms:
